### ISSUE TITLE
A11y_Model Builder_Select Scenario_Image Classification_Info and relationships: Narrator/NVDA not announcing the Sub Content that are present under "Local (CPU), "Local (GPU)", "Azure" tab items.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2668]
### GitHub Tags
#WCAG1.3.1; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Info and relationships; #Narrator; #NVDA; #STP; #Screen reader;
### Actual Result
Narrator/NVDA not announcing the Sub Content that are present under "Local (CPU), "Local (GPU)", "Azure" tab items.
Note: Issue is also observed with NVDA.

**Note: We are facing similar issue throughout the application**

### Expected Result
Narrator/NVDA should announce the Sub Content that are present under "Local (CPU), "Local (GPU)", "Azure" tab items.

### Environment Details
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755
Narrator/NVDA 2023.0.1

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
6. Tab till Add and then tab till Machine Learning Model and press enter.
7. Model Builder dialog will open. Tab till Scenario tab.
8. Select Scenario screen will appear.
9. Tab till "Image Classification" and press enter.
10. Now turn on Narrator/NVDA.
11. Tab till Tab items "Local (CPU), "Local (GPU), "Azure".
12. Now verify that Narrator/NVDA not announcing the Sub Content that are present under "Local (CPU), "Local (GPU)", "Azure" tab items.

### User Impact
Screen reader users will not be able to understand the exact content what that tab items refers to.

### Attachment
![Sub content_Narrator](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/cd15f64f-3410-4841-9ea9-e6b00a992084)

NVDA
![Sub content_NVDA](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/83544a6f-726d-4045-be71-c156af9a8857)
Code snippet:
![Bug_Sub content_Code snippet](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/1a5cdaf7-afba-499a-9abc-cc39cdf2bb85)
**Similar issue Attachment**
Narrator
![Bug_Nextsteps_Narrator](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/b24976a3-f046-44b5-a86a-58f9a97ab8c3)
NVDA
![Bug_Nextsteps_NVDA](https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/fb592180-9317-46dd-9256-562b48a4f9a3)

**Comment by Harika1702** [comment]
@zewditu 
Keyboard focus should not go to the sub content with arrow keys because it is a non-interactive element text. But here whenever Screen reader focus lands on the specific section Screen reader should announce all the content along with the label that present in it.
#Regressed:06-12-23;

**Comment by zewditu** [comment]
@Harika1702 , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by Yogi2204** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
![Recording-20230629_175303](https://github.com/dotnet/machinelearning-modelbuilder/assets/130456851/f185ca4c-0ec2-4d0a-8dfc-c16ffb08bcae)
![Recording-20230629_175123](https://github.com/dotnet/machinelearning-modelbuilder/assets/130456851/6bef1705-847d-4824-9e12-1c42c49ecde6)
